config.example.yaml

```diff
@@ -28,10 +28,11 @@ db:
   datastore:
     # enable datastore debug mode
     debug: false
-    # datastore engine - sqlite/postgres/mysql/mongodb
-    engine: sqlite
+    # datastore engine - sqlite/postgresql/mysql/mongodb
+    engine: postgres
```

Comment:
I think it's not correct - IMO it should be:
    # datastore engine - sqlite/postgresql/mysql/mongodb (experimental)
    engine: postgresql

Reply:
We should still have sqlite as default option

Reply:
Thanks. Will fix